src/core/Form/SearchInput/SearchInput.tsx

```diff
@@ -19,31 +21,45 @@ const StyledTextInput = styled(
   ({
     tokens,
     className,
+    labelTextProps = { className: undefined },
+    inputContainerProps = { className: inputContainerBaseClassName },
     inputClassName,
     ...passProps
   }: TextInputProps & InternalTokensProp) => (
-    <TextInput
+    <CompSearchInput
+      {...passProps}
+      labelTextProps={{
+        ...labelTextProps,
+        className: classnames(
+          labelTextProps.className,
```

Comment:
Am I reading this wrong or is this set as undefined above? If so, could this line be removed?

Reply:
Above uses spread defaults. It checks that if `labelTextProps` not defined - className is undefined.
It could also use just empty object as default.